pkg/template/crud/list.go

```diff
@@ -0,0 +1,37 @@
+// List file for {{.Singular}} instances
+
+package crud
+
+import (
+	"github.com/aerogear/charmil/core/factory"
+	"github.com/spf13/cobra"
+)
+
+type listOptions struct {
+	// Add your option fields here
+
+}
+
+// NewListCommand creates a new command for listing instances.
+func NewListCommand(f *factory.Factory) *cobra.Command {
+	opts := &listOptions{}
+
+	cmd := &cobra.Command{
+		Use:     f.Localizer.LocalizeByID("crud.cmd.list.use"),
```

Comment:
are we creating those localizations?

Reply:
Yes, I had added a locale file for this initially but then removed it later through a different commit.
As mentioned [over here](https://github.com/aerogear/charmil/pull/171#:~:text=Think%20of%20a%20way%20to%20generate%20locale%20file%20for%20the%20generated%20CRUD%20files%20(Blocked%20by%20the%20implementation%20of%20charmil%20init%20command.%20Can%27t%20be%20resolved%20at%20this%20moment)), the location where the locale file should be generated, depends upon the `charmil init` command implementation.

I'm creating an issue for this and will resolve it through a separate PR once the `charmil init` part is completed.

Reply:
Lets add that. Wuthout it there is not much value in those

Reply:
my prefference will be to have single yaml file

Reply:
find the locales directory and add locales there.
It will be present in /cmd/cliname/locales/....

Reply:
> find the locales directory and add locales there.
> In a cli, It will be present in /cmd/cliname/locales/....

Yes, this is exactly what I wanted to know about the `charmil init` implementation. Will add locale file according to this.
Thanks :)